EventCatalog/Controllers/CatalogController.cs

```diff
@@ -170,5 +170,24 @@ public async Task<IActionResult> UpdateEvent([FromBody] EventsCatalog eventObj)
             }
             return BadRequest();
         }
+
+        //Delete an Event method
+        [HttpDelete]
+        [Route("[action]")]
+        public async Task<IActionResult> delete([FromBody] EventsCatalog eventObj)
```

Comment:
Instead of taking whole EventsCatalog object, we can take just take id. As per my thinking, client has to create EventCatalog object everytime. I think for delete and get only Id is sufficient.

Reply:
This actually depends upon if you want to delete an item by id or pass the object.
If we pass the Id as a parameter, then we add an extra step to find the object by that Id and again pass the same object to Remove function which otherwise we would have done directly. Another option is to use overloading to have an option of either id or object.
Anyways I will change that to id.

Reply:
Changed.